barretenberg/cpp/src/barretenberg/vm2/constraining/recursion/recursive_verifier.test.cpp

```diff
@@ -205,9 +207,9 @@ TEST_F(AvmRecursiveTests, GoblinRecursion)
     outer_circuit.ipa_proof = convert_stdlib_proof_to_native(verifier_output.ipa_proof);
 
     // Ensure that the pairing check is satisfied on the outputs of the recursive verifier
-    auto pcs_vkey = std::make_shared<typename RecursiveFlavor::VerifierCommitmentKey>();
+    NativeVerifierCommitmentKey pcs_vkey{};
```

Comment:
{} for initialization is generally not needed (unless you do something weir like we do with Fr* or like how std::array works) and it's discouraged.

```
NativeVerifierCommitmentKey pcs_vkey;
```
should work and in general the same is true for any type. Things that sometimes come up like
```
std::vector<T> v = std::vector<T>();
```
should equally be avoided. I'm sure you know but in C++ unless you use ptrs all types are either default initialized or will not compile :)

*at some point we need to talk about Fr's default non-init but it's not a priority :)

Reply:
Documenting my discussion with Facundo on Slack for posterity: TLDR modern guidelines are somewhere between neutral and positive on brace initialization. In this particular case the two are equivalent and it comes down to personal preference.